[![Work in Repl.it](https://classroom.github.com/assets/work-in-replit-14baed9a392b3a25080506f3b7b6d57f295ec2978f6f33ec97e36a161684cbe9.svg)](https://classroom.github.com/online_ide?assignment_repo_id=362139&assignment_repo_type=GroupAssignmentRepo)
# solo_115 - *E. coli* response genes to antibiotic stress under ampicillin.

## Milestones

Details for Milestone are available on Canvas (left sidebar, Course Project) or [here](https://firas.moosvi.com/courses/data301/project/milestone01.html).

## Describe your topic/interest in about 150-200 words

I chose my topic on antibiotic resistance because I am highly interested in this growing global issue and hope to complete a masters degree in this area. 
Additionally, I have prior experience working with large gene datasets.

Antibiotic resistance is largely unaddressed as antibiotics are not profitable for pharmaceutical companies to produce and the topic is rather under-explored in academia. But as use of antibiotics climb resistance does too. The last novel antibiotic was produced over 40 years ago and thus, our ability to fight pathogens is time-limited. I want to be part of the solution.

For the dashboard, I’d like a graph to visualize the data and to create a search bar (if possible) so one can search for a gene and see if (according to this study) it is up/down regulated during treatment with ampicillin. Essentially my question is – what genes may be interesting to study regarding antibiotic resistance?  

I hope this project gives me valuable skills that I can apply to future research that would speed up analysis and make the work easier and efficient. I imagine similar searching capabilities that are lab specific may save time when finding genes to use compared to large cumbersome datasets.

## Describe your dataset in about 150-200 words

This dataset was provided by H. S. Deter and N. C. Butzin at the South Dakota State University in the United States of America and accessed through the NCBI Gene Expression Omnibus (GEO) online. The dataset includes genes detected by RNA-sequencing for three replicates per three conditions: 1. stationary phase cells and two populations incubated in fresh media for three hours - base line 2. no ampicillin in fresh media for three hours – untreated control 3. ampicillin applied at bactericidal concentrations in fresh media for three hours – treated variable. The data was submitted to GEO in August of 2020 and made public in December of 2020 on GEO. Datasets are uploaded to GEO to meet journal publication requirements - sequence data must be made available in a public repository. Additionally, uploading to GEO also increases the visibility, usability, and accessibility of the research. The experimental data was collected and performed by a research team and the RNA was sequenced using the Illumina HiSeq 2000 sequencing system.

## Team Members

Sam Yeo: 4th year Microbiology student currently living in Vancouver, BC. I love the ocean, microbes, and petting my cat :)

## References

Butzin, N.C., Deter H.S. "Regulatory proteins control a transcriptional network in response to antibiotic stress" Series GSE156896. GEO Accession viewer. (2020). Retrieved February 8, 2021, from https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE156896

Conly, J., & Johnston, B. (2005). "Where are all the new antibiotics? The new antibiotic paradox." The Canadian Journal of Infectious Diseases & Medical Microbiology, 16(3), 159–160. Retrieved February 8, 2021, from https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2095020/ 

Frequently Asked Questions — GEO - NCBI. (n.d.). Retrieved February 8, 2021, from https://www.ncbi.nlm.nih.gov/geo/info/faq.html
